controllers/newsletter.js

```diff
@@ -0,0 +1,46 @@
+const newsletters = require("../models/newsletter");
+
+const addEmail = async (req, res) => {
+  try {
+    const a = await newsletters.subscribe(req.body);
+    return res.json({
+      message: "gotcha",
+      response: a,
```

Comment:
What are we getting in a? Can we use better variable name here?

Reply:
done